src/main/resources/vertx-maven-plugin.properties

```diff
@@ -1,3 +1,5 @@
 maven-compiler-plugin-version=3.1
```

Comment:
actually i have this as placeholder for all default versions that will be used by the plugin.